### PR TITLE
Some Java cleanup: removal of unused imports, addition of some @NotNulls

### PR DESCRIPTION
**Pull request by jwren**
@pq

### CODE REVIEW

File: src/io/flutter/actions/RunFlutterAction.java [hunk 1]
```diff
@@ -105,10 +105,7 @@ private static boolean shouldEnable(@Nullable AnActionEvent e) {
   protected static String getSelectedRunConfig(@Nullable AnActionEvent e) {
     final RunnerAndConfigurationSettings settings = getRunConfigSettings(e);
     if (settings != null) {
-      final RunConfiguration configuration = settings.getConfiguration();
-      if (configuration != null) {
-        return configuration.getName();
-      }
+      return settings.getConfiguration().getName();
```
Comment (pq) [inline src/io/flutter/actions/RunFlutterAction.java]: Cool. Did this used to possibly return null I wonder (rhetorically).

File: src/io/flutter/sdk/FlutterSdkManager.java [hunk 2]
```diff
@@ -110,23 +110,23 @@ default void flutterSdkRemoved() {
   // Listens for changes in Flutter Library configuration state in the Library table.
   private final class LibraryTableListener implements LibraryTable.Listener {
     @Override
-    public void afterLibraryAdded(Library newLibrary) {
+    public void afterLibraryAdded(@NotNull Library newLibrary) {
```
Comment (pq) [inline src/io/flutter/sdk/FlutterSdkManager.java]: Nice.  Thanks!

File: src/io/flutter/actions/RunFlutterAction.java (hunk 1 above)
Comment (jwren) [inline src/io/flutter/actions/RunFlutterAction.java]: I didn't go look, possibly.

**Comment by pq** [comment]
FYI: it looks like this got merged into _your_ master... not upstream?

**Pull request merged**